tikvrpc/interceptor/interceptor.go

```diff
@@ -122,29 +153,49 @@ func NewRPCInterceptorChain() *RPCInterceptorChain {
 // Link is used to link the next RPCInterceptor.
 // Multiple interceptors will be executed in the order of link time.
```

Comment:
add a comment about the behavior of linking duplicated interceptor

Reply:
done